packages/hash/api/src/db/postgres/aggregation/util.ts

```diff
@@ -140,14 +142,16 @@ export const addSourceEntityVersionIdToAggregation = async (
 ) => {
   await conn.one(
     sql`
+    with updated as (
       update aggregations
       set source_entity_version_ids = array_append(aggregations.source_entity_version_ids, ${params.newSourceEntityVersionId})
       where
         source_account_id = ${params.sourceAccountId}
         and source_entity_id = ${params.sourceEntityId}
         and path = ${params.path}
         and not ${params.newSourceEntityVersionId} = ANY(aggregations.source_entity_version_ids)
-      returning *;
+      returning *
+    ) select * from updated order by created_at desc limit 1;
```

Comment:
we (@akash-joshi and I) added this while pairing to get 1 result back when multiple aggregations on the same path / source were created - @benwerner01 might be worth us discussing as I'm not sure this is the right approach

Reply:
Why would there be more than one aggregation for a single source entity at a certain `path`? I've been assuming there would only be one aggregation operation at a path, which I think is why I used `conn.one(` instead of `conn.any(` here.

Reply:
Because different aggregations can apply to the same path at different versions of an entity.

Reply:
e.g. user creates an aggregation link, deletes it, and then later creates another one later.

or deletes one and immediately creates another.

in both cases the original record isn't deleted, it just stops applying to future versions of the source entity

Reply:
If we do want to have more than one aggregation with the same source entity and `path`, we will probably need an `aggregationId` as the source entity + `path` would no longer unique identifiers for the aggregation.